bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/XToPsiNSG/bbH_HToJPsiG_JPsiToMuMu/bbH_HToJPsiG_JPsiToMuMu_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model jpsi_sm
+define p = g u c d s u~ c~ d~ s~
+generate p p > b b~ h HIG<=0 QED<=1, (h > jpsi a, jpsi > mu+ mu-)
```

Comment:
Can you please explain the meaning of `HIG<=0`? I'm not familiar with it.

Reply:
From what I remember, it refers to the total allowed power for the Higgs coupling. Also, few years ago I remember we had to add it in order to have this model to work. I might remember wrong, so I will do few more tries with this one, because ideally we don't want HIG<=0. In any case, with the run2 samples, we didn't observe issues by using this datacard(s)